js/stat.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var GAP = 10;
+var FONT_GAP = 15;
+var TEXT_WIDTH = 50;
+var BAR_HEIGHT = 150;
+var BAR_WIDTH = 40;
+
+
+var renderCloud = function (ctx, x, y, color) {
+  ctx.fillStyle = color;
+  ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var getMaxElement = function (arr) {
+  var maxElement = arr[0];
+
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] > maxElement) {
+      maxElement = arr[i];
+    }
+  }
+
+  return maxElement;
+};
```

Comment:
Вместо цикла for лучше использовать более подходящий для этого способ - метод reduce
```suggestion
var getMaxElement = function (items) {
  return items.reduce(function(acc, cur) {
    return cur > acc ? cur : acc;
  }, -Infinity)
};
```

Либо можно использовать метод Math.max, но он принимает не массив а последовательность значений. Эту проблему можно обойти с помощью метода apply
```suggestion
var getMaxElement = function (items) {
  return Math.max.apply(null, items)
};
```